internal/queries/queries.sql

```diff
@@ -342,7 +342,7 @@ WHERE
     AND seq_ns.nspname !~ '^pg_temp'
     -- Exclude sequences owned by identity columns.
     --  These manifest as internal dependency on the column
-    AND (depend.deptype IS NULL OR depend.deptype != 'i')
+    AND (depend.deptype IS null OR depend.deptype != 'i')
```

Comment:
The new version of sqlfluff converts all caps `NULL` to `null`, which ensures consistency. Other occurences of `null` in this file are also lowercase.